esp/esp/customforms/DynamicForm.py

```diff
@@ -756,6 +758,24 @@ def rebuildData(self):
             'perms': self.form.perms,
             'pages': self._getFormMetadata(self.form)
         }
+        # Identify if this form has been linked to a registration module
+        if self.form.link_type == 'Program' and self.form.link_id != -1:
+            try:
+                prog = Program.objects.get(id=self.form.link_id)
+            except Program.DoesNotExist:
+                raise ESPError('No program with ID %i' % (self.form.link_id))
+            tags = Tag.objects.filter(content_type=ContentType.objects.get_for_model(Program), object_id=prog.id, value=self.form.id)
+            if tags.count() == 1:
+                tag = tags[0]
+                if '_extraform_id' in tag.key:
+                    tl = tag.key.split("_")[0]
+                    module = "CustomFormModule"
+                elif tag.key == 'quiz_form_id':
+                    tl = "teach"
+                    module = "TeacherQuizModule"
+            else:
+                raise ESPError('Custom form # %i is linked to multiple registration modules for %s' % (self.form.id, prog.name))
```

Comment:
So I'm getting such an error in my terminal, but on the actual custom forms page on the site, nothing appears out of the ordinary. Don't we want to signal to the user that they need to fix something? Not just give an error in the log or terminal?